dbmanager/dbmanager.go

```diff
@@ -0,0 +1,116 @@
+package dbmanager
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	_ "github.com/lib/pq" //db driver for postgres
+	"golang.org/x/crypto/bcrypt"
+
+)
+
+//////// QUERIES
+const (
+	userExistsQuery  = `SELECT uid from users WHERE email = $1`
+	getUserInfoQuery = `SELECT * from users WHERE email = $1`
```

Comment:
Feels like userExists query isn't really needed and getUserInfo query can be used instead.